File: CSharp/SampleSite/Models/AuthenticationPostResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SampleSite.Models {
	public class AuthenticationPostResponse {
		public bool Success { get; set; }
		public string Message { get; set; }
		public string ValidationResults { get; set; }
	}
}

File: Java/on-premises-restricted/src/main/java/sample/models/AuthenticationPostResponse.java
package sample.models;

public class AuthenticationPostResponse {
	private boolean success;
	private String message;
	private String validationResults;

	public boolean isSuccess() {
		return success;
	}

	public void setSuccess(boolean success) {
		this.success = success;
	}

	public String getMessage() {
		return message;
	}

	public void setMessage(String message) {
		this.message = message;
	}

	public String getValidationResults() {
		return validationResults;
	}

	public void setValidationResults(String validationResults) {
		this.validationResults = validationResults;
	}
}
